src/modules/twap/state/emulatedPartOrdersAtom.ts

```diff
@@ -0,0 +1,55 @@
+import { atom } from 'jotai'
+
+import { OrderClass, OrderStatus, SigningScheme } from '@cowprotocol/cow-sdk'
+
+import { OrderStatus as OrderStatusInApp } from 'legacy/state/orders/actions'
+
+import { OrderWithComposableCowInfo } from 'common/types'
+
+import { twapOrdersListAtom } from './twapOrdersListAtom'
+import { twapPartOrdersListAtom } from './twapPartOrdersAtom'
+
+import { TwapOrderItem, TwapOrderStatus } from '../types'
+
+export const emulatedPartOrdersAtom = atom<OrderWithComposableCowInfo[]>((get) => {
+  const twapOrders = get(twapOrdersListAtom)
+  const twapParticleOrders = get(twapPartOrdersListAtom)
+
+  return twapParticleOrders.map<OrderWithComposableCowInfo>((order) => {
+    const parent = twapOrders[order.twapOrderId]
+
+    const creationDate = new Date(1 + (order.order.validTo - parent.order.t) * 1000)
```

Comment:
why +1?

Reply:
To distinguish parent and the first child :)
It's not a big deal, I can remove it